modules/3d-tiles/src/lib/tileset/tileset-3d-traverser.js

```diff
@@ -63,57 +63,16 @@ export default class Tileset3DTraverser {
   selectTile(tile, frameState) {
     tile._selectedFrame = frameState.frameNumber;
     this.result.selectedTiles.push(tile);
-
-    // if (tile.contentVisibility(frameState) !== Intersect.OUTSIDE) {
-    //   if (tile.content.featurePropertiesDirty) {
-    //     // A feature's property in this tile changed, the tile needs to be re-styled.
-    //     tile.content.featurePropertiesDirty = false;
-    //     tile.lastStyleTime = 0; // Force applying the style to this tile
-    //     tileset._selectedTilesToStyle.push(tile);
-    //   } else if (tile._selectedFrame < frameState.frameNumber - 1) {
-    //     // Tile is newly selected; it is selected this frame, but was not selected last frame.
-    //     tileset._selectedTilesToStyle.push(tile);
-    //   }
-    //   tile._selectedFrame = frameState.frameNumber;
-    //   tileset.selectedTiles.push(tile);
-    // }
   }
 
   selectDesiredTile(tile, frameState) {
-    if (!this.options.skipLevelOfDetail) {
-      if (tile.contentAvailable) {
-        // The tile can be selected right away and does not require traverseAndSelect
-        this.selectTile(tile, frameState);
-      }
-      return;
-    }
-
-    // If this tile is not loaded attempt to select its ancestor instead
-    const loadedTile = tile.contentAvailable ? tile : tile._ancestorWithContentAvailable;
```

Comment:
@loshjawrence Don't we need this logic, to show parents when children are not loaded?

Reply:
This logic is for skipLOD which requires issuing stencil gpu commands to get it to look not terrible.

Reply:
OK in that case, let's cut for now.